src/sql/workbench/parts/notebook/models/modelInterfaces.ts

```diff
@@ -520,7 +520,8 @@ export interface ICellMagicMapper {
 export namespace notebookConstants {
 	export const SQL = 'SQL';
 	export const SQL_CONNECTION_PROVIDER = 'MSSQL';
-	export const sqlKernel: string = localize('sqlKernel', 'SQL');
+	export const sqlKernel: string = 'SQL';
```

Comment:
Super nitpick here. Can we remove the ': string' here? None of the other consts have this and it's pretty clear that this is a string.